app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,25 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def job_applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @application_statuses = vacancy_applications.group_by { |k| k[:status] }.transform_values(&:count)
+  end
+
   private
 
+  def vacancy_applications
```

Comment:
Could we omit applications with draft status at this stage, to make certain that they won't be displayed anywhere if a bug is introduced in the view?

Reply:
I couldn't get into the review app so was unable to check for myself that this doesn't display draft applications.

Reply:
review app gh action was failing this aft for some reason